editor/changeset_wrapper.cpp

```diff
@@ -39,19 +37,19 @@ bool OsmFeatureHasTags(pugi::xml_node const & osmFt)
   return osmFt.child("tag");
 }
 
-string const static kVowels = "aeiouy";
+std::string_view constexpr static kVowels = "aeiouy";
 
-vector<string> const static kMainTags = {"amenity",   "shop",    "tourism", "historic", "craft",
-                                         "emergency", "barrier", "highway", "office",   "leisure",
-                                         "waterway",  "natural", "place",   "entrance", "building"};
+auto constexpr static kMainTags = {"amenity",   "shop",    "tourism", "historic", "craft",
+                                   "emergency", "barrier", "highway", "office",   "leisure",
+                                   "waterway",  "natural", "place",   "entrance", "building"};
 
-string GetTypeForFeature(XMLFeature const & node)
+std::string GetTypeForFeature(XMLFeature const & node)
 {
-  for (string const & key : kMainTags)
+  for (std::string const & key : kMainTags)
```

Comment:
strings will be created each function call from initializer_list<char const *>

Make std::string kMainTags[] = {} above.